proto/ocpaths.proto

```diff
@@ -76,3 +80,39 @@ message OCPathConstraint {
     string platform_type = 1;
   }
 }
+
+// GNMIRpc describes expected (or supported) behavior for a particular
+// Openconfig path.
+message GNMIRpc {
+  bool get = 1;
```

Comment:
I have heard requests to have some explicit negation of a required oc path.  ie: `bool not_supported = 4;` (or not_used, not_required, etc...)

The use case is to explicitly mark some path is not required which may be useful when:
1.  There is a history where the path was previously supported, but no longer is. 
2. There is uncertainty if the path in question was accidentally omitted from the list.

Reply:
That seems reasonable, overall. I just don't like having negative parameters. (_flashback to `disable no_zero_suppression false`_)

Perhaps we can lump get/set/subscribe into an enum?

```
  // Support allows specification for whether a path is required / supported for each of Get/Set/Subscribe.
  enum Support {
    UNSPECIFIED_SUPPORT = 0;
    UNSUPPORTED = 1;  // This path explicitly is not required / the target has no support for this path.
    GET = 2;  // Get requests must / do succeed for this path.
    SET = 3;  // Set requests must / do succeed for this path.
    SUBSCRIBE = 4;  // Certain subscribe requests must / do succeed for this path, as outlined below.
  }
  repeated Support support = 1;
```

WDYT?

Reply:
Ok, turns out the idea for a 'not supported' case is no longer a use case. 

Let's proceed with what you have here in this PR.   No need to add in the enum.